apollo-router/src/warp_http_server_factory.rs

```diff
@@ -276,6 +276,7 @@ where
         })
 }
 
+#[tracing::instrument(skip_all, name = "graphql_request", fields(query = %request.query, operation_name = %request.operation_name.clone().unwrap_or_else(|| "-".to_string()), client_name, client_version), level = "info")]
```

Comment:
:yarn: avoid exceeding 100 chars. Cargo format won't enforce that for macros, you have to do it manually.

```suggestion
#[tracing::instrument(
    skip_all,
    name = "graphql_request",
    fields(
        query = %request.query,
        operation_name = %request.operation_name.clone().unwrap_or_else(|| "-".to_string()),
        client_name,
        client_version
    ),
    level = "info"
)]
```

Reply:
done: 25ad732bbc67f5fdaf5e02304aaccd915dd8454f